modules/use-global-search.adoc

```diff
@@ -7,13 +7,20 @@
 
 By using global search you can search across all resources in your environment. Based on the resource type you use in your search query, the results are grouped in the following categories:
```

Comment:
You can decide the pro and con of listing the categories, because once in a while categories are quietly added.

For example, **Policy categories** has been added in 3.74 although the category has results only when product is installed with postgress database (which is standard for Cloud Services customers, but not yet standard for other customers).

Reply:
A comment for future consideration, not a requested change for now: **resources** as a generic term has room for improvement, because the term has specific meanings in Kubernetes:
* Resource types: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-types
* Resource units: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes
* Custom resources: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/

Reply:
Thank you @pedrottimark I've created https://issues.redhat.com/browse/ROX-15099 to track this suggestion.